ios/RCTWebRTC/WebRTCModule.m

```diff
@@ -22,6 +22,11 @@ @implementation WebRTCModule
 
 @synthesize bridge = _bridge;
 
++ (BOOL)requiresMainQueueSetup
```

Comment:
we don't care about alphabetic order in this file, do we ?

Reply:
I don't think we do. But this is a class method, which usually go above instance methods, I think.